Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkManager.java

```diff
@@ -349,6 +349,25 @@ public INetwork getNetwork(String networkId) throws QuadrigaStorageException {
         }
         return networkList;
     }
+    
+    /**
+    * 
+    * {@inheritDoc}
+    * 
+    * This implementation uses the hibernate for dataaccess from the database
+    */
+    @Override
+    @Transactional
+    public List<INetwork> getApprovedNetworkList() throws QuadrigaStorageException {
+        List<INetwork> networkList = new ArrayList<INetwork>();
+
+        try {
+            networkList = networkmapper.getListOfApprovedNetworks();
+        } catch (QuadrigaStorageException e) {
+            logger.error("Something went wrong in DB", e);
```

Comment:
the method signature says it throws an exception but here you just log it and don't throw anything.